__tests__/__resources__/properties/default_properties.yaml

```diff
@@ -49,6 +49,20 @@ zosmf:
   # such as a base path exposed from a gateway/ API mediation layer
   basePath: null
 #-------------------------------------------------------------------------#
+#  Connection details/credentials for Base Profile / API ML               #
+#-------------------------------------------------------------------------#
+base:
+  # your Base User ID
+  user: base-user
+  # your Base password
+  pass: base-password
+  # host for your base / API ML instance
+  host: base-host
+  # port for your base / API ML instance
+  port: base-port
+  # false to allow self signed certificates when connecting
+  rejectUnauthorized: false
```

Comment:
Contributors of the CLI will be required to add these properties to their custom_properties.yaml file if they run integration and/or systems tests if they want to try base profile stuff in their tests.

Reply:
That is correct. I don't think you were looking for a comment here, but you are getting one. 😄